django_tables2/rows.py

```diff
@@ -205,24 +213,91 @@ def items(self):
             yield (column, self.get_cell(column.name))
 
 
+class BoundPinnedRow(BoundRow):
+
+    @property
+    def attrs(self):
+        '''
+        Return the attributes for a certain pinned row.
+        Add 'pinned-row' to css class attribute
+        '''
+        row_attrs = computed_values(self._table.pinned_row_attrs, self._record)
+        cssClass = "pinned-row"
+        cssClass = " ".join([
+            self.get_even_odd_css_class(),
+            cssClass,
+            row_attrs.get('class', "")
+        ])
+        row_attrs['class'] = cssClass
+        return AttributeDict(row_attrs)
+
+    def __iter__(self):
+        '''
+        Iterate over the rendered values for cells in the row.
+
+        Under the hood this method just makes a call to
+        `.BoundPinnedRow.__getitem__` for each cell.
+        '''
+        for column, value in self.items():
+            # this uses __getitem__, using the name (rather than the accessor)
+            # is correct – it's what __getitem__ expects.
+            yield value
+
+    def _get_and_render_with(self, name, render_func):
+        '''
+        Get raw value from record render
+        this value using by render_func
+        '''
+        accessor = A(name)
+        value = accessor.resolve(context=self._record, quiet=True) or self._table.default
+        return value
+
+
 class BoundRows(object):
     '''
     Container for spawning `.BoundRow` objects.
 
     Arguments:
         data: iterable of records
         table: the `~.Table` in which the rows exist
+        pinned_data: dictionary with iterable of records for top and/or
+         bottom pinned rows. Example
+         {
+         'top' : [ iterable structure ] | None,
+         'bottom' : [ iterable structure ] | None
+         }
 
     This is used for `~.Table.rows`.
     '''
-    def __init__(self, data, table):
+    def __init__(self, data, table, pinned_data=None):
         self.data = data
         self.table = table
+        self.pinned_data = pinned_data or {}
```

Comment:
Dictionary is mutable, so maybe it will be better to use:
```
self.pinned_data = pinned_data.copy() or {}
```
to avoid changes `pinned_data` outside `__init__` method.